qiita_db/test/test_commands.py

```diff
@@ -563,7 +563,7 @@ def test_update_preprocessed_data_from_cmd(self):
         # We need to sort the list returned from the db because the ordering
         # on that list is based on db modification time, rather than id
         obs_fps = sorted(ppd.get_filepaths())
-        self.assertEqual(obs_fps, exp_fps)
+        self.assertEqual(obs_fps, sorted(exp_fps))
```

Comment:
Other checks in this tests depend on these 2 lists being in the same order. Leaving the sort.